src/main/resources/api/data-repository-openapi.yaml

```diff
@@ -3664,6 +3676,9 @@ components:
           description: Project id of the snapshot data project
         accessInformation:
           $ref: '#/components/schemas/AccessInfoModel'
+        secureMonitoringEnabled:
```

Comment:
Is this already returned with the dataset sources information in the snapshot response? In that case, why list this twice?

Reply:
I think its important to have this pointed out at the top-level of any response. Yes, the data is duplicated, but the value is populated based on the dataset sources, so its not actually being stored twice. I think visibility of the field overrides DRY-ness in this instance, but I may also be over-anticipating how much users will care about this feature.

Reply:
I see! Personally, I don't think it adds much value because the snapshot response isn't that large but I don't feel strongly about it either way

Reply:
I'm kinda with Saman on this. It seems inconsistent with how we do things in other areas (like with storage).  We don't pull that to the top object so it seems a little arbitrary that we would do it here too.  I'd lean towards just keeping it at the dataset level (especially if we add an update endpoint...which I'd love for use to do at some point for things like descriptions, etc.) and it might be confusing to have inferred values there